### ISSUE TITLE
Dashboard: DNA Sampled chart not showing "reportable" results

### ISSUE THREAD

**Issue by clegarde** [spawaskar-cora/cora-docs#321]
I added "Reportable" to "Results Confidence" for 4 skeletal elements under Seq 172 on Monday to see if the DNA Sampled chart on the Dashboard to ensure the chart was working. It's still showing 100% for "Pending".
![screen shot 2018-12-12 at 11 35 19 am](https://user-images.githubusercontent.com/7350474/49887621-7a4c2380-fe02-11e8-9841-6dc16bfd88d6.png)

I just added Inconclusive, Cancelled, and Unable to Assign each for 3 skeletal elements (9 total entries) so that this could be tested further. I know this dashboard chart only updates once per day so these are not currently reflected.

**Comment by clegarde** [comment]
@SachinPawaskarUNO, regarding your comment about this updating daily to it is working correctly: I opened this issue because it did not update with the information I added on Monday. I made sure to wait to see if the changes were reflected and they were not. I only added the other information today so that we'd know if it was only an issue with "Reportable" or with all.
@fedamann

**Comment by clegarde** [comment]
Reportable is showing correctly.

@SachinPawaskarUNO 
Pending needs to be calculated from total sampled minus reportable minus inclusive minus canceled minus unable to assign.

**Comment by SachinPawaskarUNO** [comment]
@clegarde When I looked at the code, I recollected that the business rule is that you the Anthropologist would select the Received Date and then select one of the Results Reportable option. So if you follow the rule and go back to the ones you marked as reportable and put in a Receive Date for when you received the results. It will be reflected correctly on the Dashboard. 
I however think that we should in the future make the Results Reportable dropdown only available when the user puts a valid Received Date. Maybe we can either create a new issue or use this and include a UI enhancement in the future.
@clegarde For now, I still want you to verify what Im saying above by adding a valid Received Date for those records.

**Comment by clegarde** [comment]
Received date added. Will verify issue after dashboard updates tonight.

**Comment by clegarde** [comment]
Appears to be working appropriately. The numbers don't add up, but I must not have put the date for all of the samples I added yesterday. I have to look up each one to know whether or not I did. This is a reason for adding "Results Confidence" as a way to generate an mtDNA Report. Then I could quickly generate a report for all the "Canceled" elements, for example. Currently, there's no way to know which elements those 3 are. @SachinPawaskarUNO

**Comment by clegarde** [comment]
I just checked (because I had written down the samples I added), and I had not put a date for the "Cancelled" elements. I just added this. During my check, I noticed that we've spelled cancelled/canceled two different ways.
Dashboard: Canceled
![screen shot 2018-12-13 at 11 18 52 am](https://user-images.githubusercontent.com/7350474/49955695-193b5300-fec9-11e8-8495-d658ae0554d7.png)

Results Confidence: Cancelled
![screen shot 2018-12-13 at 11 18 11 am](https://user-images.githubusercontent.com/7350474/49955715-21938e00-fec9-11e8-9bbd-352201d489d0.png)

*Not sure if this will cause any issues, but we may want to choose one way or the other.
